src/NHSD.BuyingCatalogue.Infrastructure/ValueObject.cs

```diff
@@ -4,18 +4,18 @@
 namespace NHSD.BuyingCatalogue.Infrastructure
 {
     /// <summary>
-    /// Represents a building block of a domain model. It should be a small concept representing the value of an entity.
+    ///     Represents a building block of a domain model. It should be a small concept representing the value of an entity.
```

Comment:
Is the indent a part of the Resharper code clean up?

Reply:
Yes